testing_and_setup/compass/utility_scripts/LIGHTparticles/streams.ocean

```diff
@@ -68,7 +68,7 @@
 
 <stream name="lagrPartTrackOutput"
         runtime_format="single_file"
-        filename_interval="01-00-00_00:00:00"
+        filename_interval="00-01-00_00:00:00"
         clobber_mode="truncate"
         reference_time="0001-01-01_00:00:00"
         output_interval="00-00-02_00:00:00"
```

Comment:
@bradyrx, do you want to resolve #17 and add your suggestions here?

Reply:
Should be fixed via `82156c2c882a467ada476f7bcef27ac7f78a74c1`.

Reply:
Yes, will add streams suggestions here.